awesome.c

```diff
@@ -686,6 +686,7 @@ main(int argc, char **argv)
         bool success = true;
         /* Get the first config that will be tried */
         const char *config = luaA_find_config(&xdg, confpath, true_config_callback);
+        fprintf(stdout, "Checking config %s... ", config);
```

Comment:
```suggestion
        fprintf(stdout, "Checking config '%s'... ", config);
```
(No idea if this makes sense or not)